apps/vr-tests-react-components/src/stories/InfoLabel.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 9 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27118&commitId=3f2c05f651ca523f1e2c7d849fc74a2bd89b1b02&env=prod)

✅ There was 9 screenshots added, 0 screenshots removed, 1868 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.default - Dark Mode.rest.chromium.png|0| Added|
|InfoLabel.default - High Contrast.rest.chromium.png|0| Added|
|InfoLabel.default - RTL.rest.chromium.png|0| Added|
|InfoLabel.default.rest.chromium.png|0| Added|
|InfoLabel.required - RTL.rest.chromium.png|0| Added|
|InfoLabel.required.rest.chromium.png|0| Added|
|InfoLabel.size-large.rest.chromium.png|0| Added|
|InfoLabel.size-small.rest.chromium.png|0| Added|
|InfoLabel.wrap.rest.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>